nvm.sh

```diff
@@ -1581,6 +1581,9 @@ nvm_install_binary() {
   local TMPDIR
   local VERSION_PATH
 
+  local NODE_OR_IOJS
+  [ "${FLAVOR}" = "node" ] && NODE_OR_IOJS="${FLAVOR}"
+  nvm_echo "Start to download and install ${NODE_OR_IOJS} ${VERSION}"
```

Comment:
Also, while we're making changes, let's have this copy be "Downloading and installing ${NODE_OR_IOJS-} ${VERSION}..." :-)